fern/openai-realtime.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: OpenAI Realtime
+subtitle: You can use OpenAI's newest speech-to-speech model with your Vapi assistants.
+slug: openai-realtime
+---
+
+OpenAI's Realtime API enables developers to create low-latency, voice-driven interactions within applications by leveraging a speech-to-speech model. This API processes audio input and output in real-time, supporting seamless, natural conversations without intermediate text conversion.
+
+To start using it with your Vapi assistants, select `gpt-4o-realtime-preview-2024-10-01` as your model. 
+- Please note that only OpenAI voices may be selected while using this model. The voice selection will not act as a TTS (text-to-speech) model, but rather as the voice used within the speech-to-speech model.
+
+<Note>
+  The Realtime API is currently in beta, and not recommended for production use by OpenAI. We're excited to have you try this new feature and welcome your [feedback](https://discord.com/invite/pUFNcf2WmH) as we continue to refine and improve the experience.
+</Note>
```

Comment:
nit: could move banner to top since people will already know what Realtime is and I think it would be nice to warn upfront